spyt-package/src/main/python/spyt/conf.py

```diff
@@ -26,6 +18,71 @@
 logger = logging.getLogger(__name__)
 
 
+class SpytDistributions:
+    def __init__(self, client: YtClient, yt_root: str):
+        self.client = client
+        self.yt_root = YPath(yt_root)
```

Comment:
I'd rename `yt_root` to `base_path` so it would be `SpytDistribution.base_path`, I think it makes more sense.

Reply:
It is called root in the publisher scripts: https://github.com/ytsaurus/ytsaurus-spyt/tree/382ae1656208601175b6f004fd31a39affb17b83/tools/release/publisher
Also it is not just any root or path, but a YT one.

Reply:
But I think `base_path` in the context of this class is more suitable

Reply:
In context of this class – maybe, but in context of public interface I believe that this should be coherent with the `--root` parameter from publisher scripts.